docs/src/main/paradox/reference.md

```diff
@@ -726,6 +726,46 @@ Merges three bands into a single byte-packed RGB composite. It first scales each
     
 Runs [`rf_rgb_composite`](reference.md#rf-rgb-composite) on the given tile columns and then encodes the result as a PNG byte array.
 
+### rf_render_color_ramp_png
+
+    Array rf_render_png(Tile tile, String color_ramp_name)
+    
+Converts given tile into a PNG image, using a color ramp of the given name to convert cells into pixels. `color_ramp_name` can be one of the following:
+
+ * "BlueToOrange"
+ * "LightYellowToOrange"
+ * "BlueToRed"
+ * "GreenToRedOrange"
+ * "LightToDarkSunset"
+ * "LightToDarkGreen"
+ * "HeatmapYellowToRed"
+ * "HeatmapBlueToYellowToRedSpectrum"
+ * "HeatmapDarkRedToYellowWhite"
+ * "HeatmapLightPurpleToDarkPurpleToWhite"
+ * "ClassificationBoldLandUse"
+ * "ClassificationMutedTerrain"
+ * "Magma"
+ * "Inferno"
+ * "Plasma"
+ * "Viridis"
+ * "Greyscale2"
+ * "Greyscale8"
+ * "Greyscale32"
+ * "Greyscale64"
+ * "Greyscale128"
+ * "Greyscale256"
+
+Further descriptions of these color ramps can be found in the [Geotrellis Documentation](https://geotrellis.readthedocs.io/en/latest/guide/rasters.html#built-in-color-ramps).
+
+### rf_agg_overview_raster
+    
+    Tile rf_agg_overview_raster(Tile proj_raster_col, int cols, int rows, Extent aoi)
+    Tile rf_agg_overview_raster(Tile tile_col, int cols, int rows, Extent aoi, Extent tile_extent_col, CRS tile_crs_col)
+
+Construct an overview raster of size `cols`x`rows` where data in `proj_raster` intersects the `aoi` bound box in web-mercator. Uses bi-linear sampling method.
```

Comment:
Let's be a little more verbose. It all runs together a little bit here. Is the `aoi` expressed in web mercator? IS the overview itself in the `proj_raster_col.crs` or `tile_crs_col`; or is it in web mercator? 

Give a sentence about the second set of args as well.